### DISCUSSION TITLE
sponsor

### DISCUSSION THREAD

**Discussion by new-789** [elementary/os#709] [Q&A] [answered]
Because I often use ElementaryOS, I also recognize the ease of use of this operating system, and I am interested in becoming a sponsor of this project. However, GITHUB does not support payment methods in mainland China, so how should I complete my sponsorship?

**Comment by sudanchapagain** [comment, marked as answer]
look into the following

- <https://elementary.io/get-involved#funding>
- <https://store.elementary.io/>
- the purchase option in homepage <https://elementary.io/>
